quirks/password-rules.json

```diff
@@ -141,6 +141,9 @@
     "metlife.com": {
         "password-rules": "minlength: 6; maxlength: 20;"
     },
+    "myaccess.dmdc.osd.mil": {
+        "password-rules": "minlength: 9; maxlength: 20; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789@_#!&$`%*+()./,;~:{}|?>=<^[]'-];"
```

Comment:
This isn’t a valid rule, because a `]` has to be the last character in a character class. Which gives us:

```
minlength: 9; maxlength: 20; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789@_#!&$`%*+()./,;~:{}|?>=<^'-[]];
```

But the fact the lower, upper, and digit are required means that they don’t need to be stated in allowed. Which gives us:

```
minlength: 9; maxlength: 20; required: lower; required: upper; required: digit; allowed: [@_#!&$`%*+()./,;~:{}|?>=<^'-[]];
```

And just to be sure, all of those symbols are allowed, and no symbols is required, right?